.github/workflows/continuous-integration-workflow.yml

```
@@ -109,18 +81,23 @@ jobs:
       - name: Build
         run: |
           ccache -z
-          cmake --build builddir --parallel 2
+          cmake --build builddir --parallel 2 --target KokkosCore_PerformanceTest_Benchmark
           ccache -s
       - name: Tests
         working-directory: builddir
-        run: ctest --output-on-failure
-      - name: Test DESTDIR Install
-        run: DESTDIR=${PWD}/install cmake --build builddir --target install && rm -rf ${PWD}/install/usr && rmdir ${PWD}/install
-      - name: Install
-        run: sudo cmake --build builddir --target install
-      - name: Test install
-        working-directory: example/build_cmake_installed
-        run: |
-          cmake -B builddir -DCMAKE_CXX_COMPILER=${{ matrix.cxx }}
-          cmake --build builddir
-          cmake --build builddir --target test
+        run: ctest --output-on-failure -R KokkosCore_PerformanceTest_Benchmark
+      - name: Gather benchmark results
+        run: |-
+          mkdir ${{ env.BUILD_ID }}
+          find builddir/core/perf_test/ -name "*.json" -exec mv {} ${{ env.BUILD_ID }}/  \;
+      - name: Push benchmark results
+        if: github.ref == 'refs/heads/performance-results-visualization'
```

FIXME before merging:
```suggestion
        if: github.ref == 'refs/heads/develop'
```